crates/ln-dlc-node/src/channel.rs

```diff
@@ -88,6 +118,7 @@ pub struct Channel {
     /// the funding transaction. We use the `user_channel_id` as identifier over the entirety
     /// of the channel lifecycle.
     pub channel_id: Option<ChannelId>,
+    pub fake_scid: Option<FakeScid>,
```

Comment:
🔧 We could comment that this is only `Some` if we are dealing with a JIT channel.